proto/buf.lock

```diff
@@ -0,0 +1,8 @@
+# Generated by buf. DO NOT EDIT.
+version: v1
+deps:
+  - remote: buf.build
+    owner: gogo
+    repository: protobuf
+    commit: 4df00b267f944190a229ce3695781e99
```

Comment:
Could we have this match api/proto/buf.lock?

https://github.com/gravitational/teleport/blob/97c3757f3f9b19f9671f9d9362ad84bd8b1d00fc/api/proto/buf.lock#L4-L7

Reply:
Bump? Was this addressed?

Reply:
Nevermind, there's no lock file anymore.